### DISCUSSION TITLE
Intermittent errors with HASS addon for the past week

### DISCUSSION THREAD

**Discussion by ross-w** [amberelectric/public-api#213] [Home Assistant]
I've been using the Amber HASS plugin for many years now and it has generally been rock solid.

However, since the 25th of June, I've been getting intermittent errors with the data not showing up. HASS will log
```
 Error fetching amberelectric data: Missing price data, skipping update 
 ```
 And the states of the sensors will all show "Unknown". Usually, a minute later, the state will return to normal, so I don't think it's the same issue others have reported on here where the sensor stays broken.

This is playing havoc with my automations as each time it happens it triggers things to switch on or off, when they should not have. It's happened 38 times since the 25th, the last one being this morning at 10:50 AEST.

I'm on Jemena, VIC, with tariffs A100 for both feed in and anytime.

**Comment by madpilot** [comment]
Thanks, I'll have a look.
